tests/acceptance/features/apiNotification/shareExpireNotification.feature

```diff
@@ -0,0 +1,29 @@
+@notification @email
+Feature: Share Expiry Notification
+  As a user
+  I want to be notified when share expires
+  So that I can stay updated about the share
+
+  Background:
+    Given these users have been created with default attributes:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  Scenario: check share expired in-app and mail notification for Personal space resource
+    Given using SharingNG
+    And user "Alice" has uploaded file with content "hello world" to "testfile.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource           | testfile.txt         |
+      | space              | Personal             |
+      | sharee             | Brian                |
+      | shareType          | user                 |
+      | permissionsRole    | Viewer               |
+      | expirationDateTime | 2025-07-15T14:00:00Z |
+    When user "Alice" expires the last share of resource "testfile.txt" inside of the space "Personal"
+    Then the HTTP status code should be "200"
+    And user "Brian" should get a notification with subject "Membership expired" and message:
+      | message                           |
+      | Access to Space Alice Hansen lost |
```

Comment:
>Access to Space Alice Hansen lost

the message doesn't seem to align with the action. It should similar to `Access to testfile.txt expired`

Reply:
did you forget our last discussion about message this current behavior,. I have raise question on bug report but no reply came so assuming this current behavior scenario has been written if you want perfect message then plz tell developer in standup .

Reply:
then we should not merge this PR until the expected behavior is confirmed